.gitignore

```diff
@@ -227,3 +227,10 @@ test/.jqwik-database
 .flattened-pom.xml
 /*/.flattened-pom.xml
 tags
+
```

Comment:
Why do all those files need to be added in gitignore?

Reply:
These will be removed soon. Added them for patching while testing.